android/app/src/bt/java/covidsafepaths/bt/exposurenotifications/nearby/ExposureNotificationBroadcastReceiver.java

```diff
@@ -0,0 +1,22 @@
+package covidsafepaths.bt.exposurenotifications.nearby;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+
+import com.google.android.gms.nearby.exposurenotification.ExposureNotificationClient;
+
+/**
+ * Receiver registered for notifications of a potential exposure.
+ */
+public class ExposureNotificationBroadcastReceiver extends BroadcastReceiver {
+
+    @Override
+    public void onReceive(Context context, Intent intent) {
```

Comment:
When will this receiver be called?

Reply:
That's automatically called by the API when we do `exposureNotificationClient.provideDiagnosisKeys(files, config.get(), token)`.
I'll add some comments in this class to explain it